templates/travel/partials/results/results.snip.html

```diff
@@ -0,0 +1,128 @@
+<!-- Results -->
+<section class="travel-results pb1 md-pt1">
+  <div class="max-width-3 mx-auto px1 md-px2">
+    <amp-list
+      class="travel-block-list travel-results-list"
+      layout="flex-item"
+      {{> ../util/amp-list-props.snip.html}}
+    >
+      <template type="amp-mustache">
+        {{=[[ ]]=}}
+        <div class="flex flex-wrap mxn1">
+          {{#results.length}}
+            {{#results}}
+            <div class="col-12 sm-col-6 lg-col-4 p1 travel-results-result">
+              <div class="relative travel-results-result">
+                <a class="travel-results-result-link" href="#">
+                  <amp-img
+                    class="rounded bg-silver mb1"
+                    width="2"
+                    height="1"
+                    layout="responsive"
+                    src="/img/travel/activity/{{img}}.jpg"
+                    srcset="/img/travel/activity/{{img}}@2x.jpg 2x, /img/travel/activity/{{img}}.jpg 1x"
+                  ></amp-img>
+                </a>
+                {{#flags.new}}
+                <div class="travel-results-result-flags">
+                  <div class="p1"><span class="travel-pill bold">NEW</span></div>
```

Comment:
i think you only need the span, or the div. You can use m1 instead of p1 and add `block`